uitests/plugins/org.jboss.tools.fuse.reddeer/src/org/jboss/tools/fuse/reddeer/view/FusePropertiesView.java

```diff
@@ -185,27 +178,47 @@ else if (items.contains(value))
 		});
 
 	}
-	
+
 	public String getLabel(final Widget widget) {
-		
 		activate();
 		final StringBuilder builder = new StringBuilder();
 
 		Display.syncExec(() -> {
 			Label label = (Label) widget;
 			builder.append(label.getText());
 		});
-		
+
 		return builder.toString();
 	}
 
-	public Map<Widget, Widget> getDetailsProperties() {
+	public List<String> getPropertiesLabelsList(final String tab) {
+		List<String> properties = new ArrayList<>();
+		Map<Widget, Widget> widgets = getAdvancedProperties(tab);
+		for (Widget w : widgets.keySet()) {
+			if(w instanceof Label)
+				properties.add(getLabel(w));
+		}
+		/*
```

Comment:
do not leave unused Java code in code base - please remove commented-out Java code